addon/prefixer.js

```diff
@@ -43,11 +44,36 @@ exports.addon = function (renderer) {
         var result = renderer.prefix(prop, value);
 
         var returned = '';
-        Object.keys(result).forEach(function(key) {
+        Object.keys(result).forEach(function (key) {
             var str = decl(key, value);
             returned += str;
         });
 
         return returned;
     };
+
+    function newPut(selector, decls, atrule) {
+        var indexed = selector.indexOf('::placeholder');
+        if (indexed > -1) {
+            var split = selector.split(',');
+            if (split.length > 1) {
+                split.forEach(function(sp) {
+                    newPut(sp.trim(), decls, atrule);
+                });
+                return;
+            }
+            var bareSelect = selector.substring(0, indexed);
+            [
+                '::-webkit-input-placeholder',
+                '::-moz-placeholder',
+                ':-ms-input-placeholder',
+                ':-moz-placeholder'
```

Comment:
Should `::placeholder` also be in this list?

Reply:
It adds those rules if it finds a "::placeholder" rule. It doesn't replace the "::placeholder" rule, just repeats it. On line 75, it `puts` the original selector.

I added "::placeholder" to the tests to be sure it's included.

Reply:
Thanks!